guard/Makefile

```diff
@@ -42,6 +42,7 @@ LOG_LEVEL?=info
 ROOT_LOGIN?="true"
 ROOT_LOGIN_METHODS?="github"
 DEFAULT_LOGIN_METHOD?="local"
+RESTRICTED_ORG_USERNAMES?="admin,www,billing,api"
```

Comment:
let's maybe put here `www,api,id,me` admin and billing are not used on CE at all.